lib/plugins/filter/before_post_render/backtick_code_block.js

```diff
@@ -48,6 +49,14 @@ function backtickCodeBlock(data) {
           }
         }
       }
+
+      if (exclude_languages.includes(args)) {
```

Comment:
`args` may include both language and caption, should use `options.lang` instead.

Reply:
Fixed.